using_openshift/integrating_external_services.adoc

```diff
@@ -150,24 +156,33 @@ service:
 }
 ----
 
-<1> Other fields on the `DeploymentConfig` are omitted
+<1> Other fields on the `DeploymentConfig` are omitted.
+<2> `SAAS_API_KEY`: The API key to use with the service.
+<3> `SAAS_USERNAME`: The username to use with the service.
+<4> `SAAS_PASSPHRASE`: The passphrase to use with the service.
+
+====
 
 === Using the service in your application
 
-Using an external service in your application is just like using an internal service.  Your
-application will be injected with environment variables for the service and the additional
-environment variables with the credentials described above.  In our example, The 'helloworld'
-container will receive the following environment variables:
+Using an external service in your application is just like using an internal service. Your application is
+injected with environment variables for the service and the additional environment variables with the credentials
+described above. In the following example, the 'helloworld' container receives the following environment variables:
 
-----
+====
 EXAMPLE_EXTERNAL_SERVICE_SERVICE_HOST=10.0.0.11 <1>
-EXAMPLE_EXTERNAL_SERVICE_SERVICE_PORT=1234
-SAAS_API_KEY=<value of SaaS api key>
-SAAS_USERNAME=<value of SaaS username>
-SAAS_PASSPHRASE=<value of SaaS passphrase>
-----
 
-<1> Example value only; actual IP will vary
+EXAMPLE_EXTERNAL_SERVICE_SERVICE_PORT=1234 <2>
+
+SAAS_API_KEY=[replaceable]#<value of SaaS api key>#
+
+SAAS_USERNAME=[replaceable]#<value of SaaS username>#
+
+SAAS_PASSPHRASE=[replaceable]#<value of SaaS passphrase>#
+====
+
+<1> An example IP address.
+<2> An example port number.
```

Comment:
@bfallonf If you meant for these to show up as a code block within an example, it's missing some `----`s (see https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#examples). Also the `<1>` and `<2>` should be within the same example block; right now they're just showing up as plain text.